### ISSUE TITLE
laminard unresponsive after nmap port scan

### ISSUE THREAD

**Issue by arunisaac** [ohwgiles/laminar#172]
When I run nmap to scan the port laminard is listening on, laminard becomes unresponsive.

```
$ curl -I http://localhost:8080
HTTP/1.1 200 OK
Content-Length: 3134
Content-Type: text/html; charset=utf-8
Content-Encoding: gzip
Content-Transfer-Encoding: binary

$ nmap -p8080 localhost
Starting Nmap 7.92 ( https://nmap.org ) at 2022-08-22 13:48 IST
Nmap scan report for localhost (127.0.0.1)
Host is up (0.00015s latency).

PORT     STATE SERVICE
8080/tcp open  http-proxy

Nmap done: 1 IP address (1 host up) scanned in 0.10 seconds
$ curl -I http://localhost:8080
curl: (7) Failed to connect to localhost port 8080 after 0 ms: Connection refused
```

This is what I see on the shell output of laminard.

```
$ laminard
laminard version 1.1 started
taskFailed: ::read(fd, buffer, maxBytes): Connection reset by peer
```

**Comment by ohwgiles** [comment]
Please use the latest version, this was most likely fixed in #164

**Comment by arunisaac** [comment]
Thanks, I have now upgraded to version 1.2. I also upgraded the Guix
package.
https://git.savannah.gnu.org/cgit/guix.git/commit/?id=a36fb3d131e3db20926c5c374e26cb57a5c7b243
